abixen-platform-core/src/test/java/com/abixen/platform/core/infrastructure/configuration/PlatformTestSecurityConfiguration.java

```diff
@@ -14,21 +14,29 @@
 
 package com.abixen.platform.core.infrastructure.configuration;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-
-import static com.abixen.platform.common.infrastructure.util.PlatformProfiles.DEV;
-import static com.abixen.platform.common.infrastructure.util.PlatformProfiles.DOCKER;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 @Profile("test")
```

Comment:
Why it is plain text ?. Constant will be better

Reply:
Intentional - I don't want to expose this profile for tests and application developers. This is used only in particular places.